ui/src/components/RoleForm/index.jsx

```diff
@@ -108,8 +110,22 @@ export default class RoleForm extends Component {
     this.props.onRoleSave(role, roleId);
   };
 
+  handleDialogActionError = error => {
+    this.props({ dialogError: error });
+  };
+
   render() {
-    const { role, classes, isNewRole, loading } = this.props;
+    const {
+      role,
+      classes,
+      isNewRole,
+      loading,
+      dialogOpen,
+      dialogError,
+      onActionDialogClose,
+      onDialogActionError,
+      onDialogOpen,
```

Comment:
Some of these props are no longer defined like `onDialogActionError`.

Reply:
It is being used in line 282. Also I have verified for others also.